GEOSldas_App/ldas_setup

```diff
@@ -70,59 +70,62 @@ class LDASsetup:
             # sample sub-command
             # by construction, we can have
             # either: {'exeinp': False, 'batinp': 'lasgh'}    <-- 'lasgh'???
-            # or:     {'exeinp': True, 'batinp': None}
+            # or:     {'exeinp': True,  'batinp': None   }
```

Comment:
@weiyuan-jiang : Any idea what `'lasgh'` in line 72 might mean?  Should this be replaced with True?  Also, is there a difference between "False" in line 72 and "None" in line 73?

Reply:
Both are logic types and by default they are Ture.  So we should use True or False

Reply:
Edited -- I hope my changes are correct.